packages/evm/src/types.ts

```diff
@@ -210,7 +210,7 @@ export interface EVMOpts {
 
   /**
    * Allows unlimited contract sizes while debugging. By setting this to `true`, the check for
-   * contract size limit of 24KB (see [EIP-170](https://git.io/vxZkK)) is bypassed.
+   * contract size limit of 24KB (see [EIP-170](https://git.io/vxZkK)) is 170.
```

Comment:
Why did you change this?  This doesn't feel like a clear chainge to me.

Reply:
That one's a mistake, undoing

Reply:
Resolved